src/data/reducers/assets.js

```diff
@@ -50,9 +54,9 @@ export const assets = (state = [], action) => {
   let assetsList;
   switch (action.type) {
     case assetActions.request.REQUEST_ASSETS_SUCCESS:
-      return action.data.assets;
-    case assetActions.delete.DELETE_ASSET_SUCCESS:
```

Comment:
Thoughts on removing this? This was doing a removal of the asset from the table when the page knew that the asset was successfully deleted. This was before I added in the following getAssets call to refresh the assets on the page. Leaving this in would do an optimistic removal of the asset from the table data before the getAssets call returns. This would cause a bit of page thrash if the asset was the last on the page (deleted -> no content view -> refreshed view, as opposed to deleted -> refreshed view). I could leave this in, but there would have to be more logic added to prevent the page thrash, and I didn't want to increase the scope any more.

Reply:
I'm cool with removing it